app.js

```diff
@@ -92,27 +93,16 @@ function upgradeWebsocket(server) {
   });
 }
 
-var httpServer = http.createServer(app);
-httpServer.listen(config.port);
+var httpServer = http.createServer(letsencrypt.middleware(app)).listen(config.port, function() {
```

Comment:
I think it may be possible to use the alternate middleware format around L62 with something like:

```
app.use('/', letsencrypt.middleware())
```

Which means that we only need to wrap it once there, rather than in both http and https. (we still need the httpsoptions below)